x/cosmwasmpool/model/msgs.go

```diff
@@ -0,0 +1,83 @@
+package model
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/osmosis-labs/osmosis/v15/x/cosmwasmpool/types"
+	poolmanagertypes "github.com/osmosis-labs/osmosis/v15/x/poolmanager/types"
+)
+
+// constants.
+const (
+	TypeMsgCreateCosmWasmPool = "create_concentrated_pool"
+)
+
+var (
+	_ sdk.Msg                        = &MsgCreateCosmWasmPool{}
+	_ poolmanagertypes.CreatePoolMsg = &MsgCreateCosmWasmPool{}
+)
+
+func NewMsgCreateCosmWasmPool(
+	sender sdk.AccAddress,
+) MsgCreateCosmWasmPool {
+	return MsgCreateCosmWasmPool{
+		Sender: sender.String(),
+	}
+}
+
+func (msg MsgCreateCosmWasmPool) Route() string { return types.RouterKey }
+func (msg MsgCreateCosmWasmPool) Type() string  { return TypeMsgCreateCosmWasmPool }
+func (msg MsgCreateCosmWasmPool) ValidateBasic() error {
+	// TODO: add more validation.
```

Comment:
Should we track this todo?

Reply:
Adding to https://github.com/osmosis-labs/osmosis/issues/4765